shared/people/invite-friends/tab-bar-button.tsx

```diff
@@ -56,16 +55,16 @@ const InviteFriends = () => {
         </Kb.Text>
       </Kb.Box2>
     )
-    firstTooltipLine = `${inviteCounts.inviteCount.toLocaleString()} friends invited in the last 24 hours.`
   }
+  const tooltipMarkdown = <Kb.Markdown style={styles.tooltip}>{inviteCounts?.tooltipMarkdown}</Kb.Markdown>
```

Comment:
Have you checked to make sure this doesn't just show "undefined" if `inviteCounts` is `undefined`

Reply:
It's only used in situations where `inviteCounts` is set, but I can add a guard here too.

Reply:
Ah, that's OK; just wanted to make sure. I'm good with as-is!